.gitlab/functional_test/regression_detector.yml

```diff
@@ -10,11 +78,12 @@ single-machine-performance-regression_detector:
   needs:
     - job: single_machine_performance-full-amd64-a7
       artifacts: false
+    - job: single_machine_performance-regression_detector-merge_base_check
```

Comment:
potential win to be had by putting this as a pre-req for the build job as well

Reply:
True.